salt/modules/npm.py

```diff
@@ -162,41 +162,21 @@ def install(pkg=None,
             env.update({'SUDO_UID': uid, 'SUDO_USER': ''})
 
     cmd = ' '.join(cmd)
-    result = __salt__['cmd.run_all'](cmd, python_shell=True, cwd=dir, runas=runas, env=env)
+    result = __salt__['cmd.run_all'](cmd,
```

Comment:
I figure this is as good a time as any to switch this to be more vertical, but I can put it back if you'd prefer.